### PR TITLE
cpp templates in yaml files: Fix some template errors

### PR DESCRIPTION
**Pull request by dl1ksv**
# Pull Request Details

In some yaml files the cpp templates build wrong strings. They use 'value' instead of "value"
This is fixed here for three files.

## Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/gnuradio/gnuradio/blob/main/CONTRIBUTING.md).
- [x] I have squashed my commits to have one significant change per commit. 
- [x] I [have signed my commits before making this PR](https://github.com/gnuradio/gnuradio/blob/main/CONTRIBUTING.md#dco-signed)
- [ ] My code follows the code style of this project. See [GREP1.md](https://github.com/gnuradio/greps/blob/main/grep-0001-coding-guidelines.md).
- [ ] I have updated [the documentation](https://wiki.gnuradio.org/index.php/Main_Page#Documentation) where necessary.
- [ ] I have added tests to cover my changes, and all previous tests pass.

### CODE REVIEW

File: gr-blocks/grc/blocks_file_sink.block.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ templates:
 cpp_templates:
     includes: ['#include <gnuradio/blocks/file_sink.h>']
     declarations: 'blocks::file_sink::sptr ${id};'
-    make: 'this->${id} = blocks::file_sink::make(${type.size}*${vlen}, ${file}, ${append});'
+    make: 'this->${id} = blocks::file_sink::make(${type.size}*${vlen}, "no_quotes(${file})", ${append});'
```
Comment (mormj) [inline gr-blocks/grc/blocks_file_sink.block.yml:55]: I'm getting an error here with the file sink:
```
file_sink_base :error: [::open] no_quotes('/tmp/asdf.txt'): No such file or directory
```

Comment (dl1ksv) [reply gr-blocks/grc/blocks_file_sink.block.yml]: Sorry for that silly mistake of mine. I tested that, but pushed a wrong commit.
I fixed that now.

**Comment by dl1ksv** [comment]
Here is a working example. The frequency sink as an empty name while the waterfall sink is named Waterfall
cpp_gen

**Review by mormj — APPROVED** [review]
Appears to be working now.  Thanks!

**Pull request merged**